src/LibSassBuilder/Program.cs

```diff
@@ -93,11 +93,13 @@ async Task CompileFilesAsync(IEnumerable<string> sassFiles)
 				var result = SassCompiler.CompileFile(file, options: Options.SassCompilationOptions);
 
 				var newFile = fileInfo.FullName.Replace(fileInfo.Extension, ".css");
+				var newMapFile = fileInfo.FullName.Replace(fileInfo.Extension, ".css.map");
 
-				if (File.Exists(newFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newFile)).ReplaceLineEndings())
-					continue;
+				if (!(File.Exists(newFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newFile)).ReplaceLineEndings()))
+					await File.WriteAllTextAsync(newFile, result.CompiledContent);
 
-				await File.WriteAllTextAsync(newFile, result.CompiledContent);
+				if (!(string.IsNullOrEmpty(result.SourceMap) || (File.Exists(newMapFile) && result.CompiledContent.ReplaceLineEndings() == (await File.ReadAllTextAsync(newMapFile)).ReplaceLineEndings())))
```

Comment:
I wonder (just for performance) if it's really worth comparing the `SourceMap == newMapFile` here - could we get away with grouping this in the `CompiledContent == newFile` ? 

i.e. could we ever have a scenario where `CompiledContent == newFile && SourceMap != newMapFile`?

Reply:
The map references the source files. If their are file-name changes but no content
changes, than the css file stays the same, but the map file changes.
Is that assumption correct?

Reply:
I don't mean the sass file that will be directly converted to a css, but the imported dependencies, that are included in the new css file. A name change of such a file, would not change the css file, but will change the content of the map file.

Reply:
A file called `app.scss` will result in 2 files: `app.css` and `app.css.map`

Then changing that existing `app.scss` filename to `appnew.scss` will result in 2 new files: `appnew.css` and `appnew.css.map`

We don't keep track of filename changes or clean up orphaned files.
Regardless, this was an optional potential perf change, so don't think we have to worry about this too much.

Reply:
Sources:
- main.scss
Dependencies:
- tools.scss, someOtherStuff.scss
Output:
- main.css and main.css.map (which has a "sources" list referencing all the scss files)

if someOtherStuff.scss changes its name, than the generated main.css stays the same, but the main.css.map changes (because of the "sources")

Sorry for beeing anoying :)

Reply:
Ah okay thanks for elaborating, I missed that point initially.
However in that case, wouldn't `main.scss` have to change as well to update its dependency?